app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -285,12 +317,30 @@ function YourTurnContent(props: {
                       [...shownCardsInActiveTurn.keys()]
                     ).length === 0 &&
                     props.secondsLeft !== 0
+
+                  const scorings = shownCardIds.map(cardId => {
+                    const card = shownCardsInActiveTurn.get(cardId)
+                    if (card && card.startedAt && card.endedAt) {
+                      return {
+                        turn_id: props.activeTurn.id,
+                        card_id: cardId,
+                        score: card.status === ShownCardStatus.Complete ? 1 : 0,
+                        status: card.status,
+                        started_at: timestamptzNowFromDate(card.startedAt),
+                        ended_at: timestamptzNowFromDate(card.endedAt)
+                      }
+                    } else {
+                      return null
+                    }
+                  })
```

Comment:
save the score attributed to that card for this turn... eventually turn_scorings will be source of truth for "points"

and `score: card.status === ShownCardStatus.Complete ? 1 : 0,` can get different points based on the settings and card status